public/common/TracyQueue.hpp

```diff
@@ -405,6 +405,7 @@ enum class GpuContextType : uint8_t
     Direct3D12,
     Direct3D11,
     Metal,
+    CUDA,
```

Comment:
You also need to add `"CUDA"` to `GpuContextNames`, right after `"Metal"`:
https://github.com/wolfpld/tracy/blob/a03c7580b9101df3225937524e09a9a885d5464a/profiler/src/profiler/TracyView.hpp#L35-L44